packages/web/src/components/Header/Header.module.css

```diff
@@ -49,8 +47,6 @@
   color: var(--aqua);
   border: 1px solid var(--aqua);
   border-radius: 18px;
-  height: 30px;
-  width: 110px;
 }
```

Comment:
May not be part of this PR, but missing some padding between header buttons 
<img width="485" alt="image" src="https://github.com/AllenInstitute/aics-fms-file-explorer-app/assets/24441706/0c9ee4ef-a375-4d58-8852-46651d732e4d">

Reply:
I'll note this down for part 2